js/form.js

```diff
@@ -0,0 +1,31 @@
+
+const adFormType = document.querySelector('#type');
+const adFormPrice = document.querySelector('#price');
+const adFormTime = document.querySelector('.ad-form__element--time');
+
+const REALTY_MIN_PRICES = {
+  'bungalow': 0,
+  'flat': 1000,
+  'house': 5000,
+  'palace': 10000,
+};
+
+//ф-ия изменения минимальной цены в зависимости от типа жилья
+const minPriceChangeByType = () => {
+  let newValue = REALTY_MIN_PRICES[adFormType.value];
```

Comment:
Когда значение переменной подразумевается будет оставаться постоянным - лучше использовать const
```suggestion
  const newValue = REALTY_MIN_PRICES[adFormType.value];
```